packages/mask/src/plugins/Tips/SNSAdaptor/TipsEntranceDialog.tsx

```diff
@@ -0,0 +1,363 @@
+    const { loading, value: proofRes, retry: retryProof } = useProvedWallets()
+
+    useEffect(() => {
+        setHasChanged(false)
+        const walletsList = proofRes
+            ? (proofRes as NextIDPersonaBindings).proofs.filter((x) => x.platform === NextIDPlatform.Ethereum)
+            : []
+        walletsList.sort((a, b) => Number.parseInt(b.last_checked_at, 10) - Number.parseInt(a.last_checked_at, 10))
+        walletsList.forEach((wallet, idx) => (wallet.rawIdx = walletsList.length - idx))
+        if (kv?.ok && kv.val.proofs.length > 0 && walletsList.length > 0) {
+            const kvCache = (kv.val as NextIdStorageInfo).proofs.find(
+                (x) => x.identity === currentPersona?.publicHexKey,
+            )
+            if (!kvCache) return
+            const result: BindingProof[] = walletsList.reduce<BindingProof[]>((res, x) => {
+                x.isDefault = 0
+                x.isPublic = 1
+                const temp = (kvCache?.content[PluginId.Tip] as BindingProof[]).filter((i) =>
+                    isSameAddress(x.identity, i.identity),
+                )
+                if (temp && temp.length > 0) {
+                    x.isDefault = temp[0].isDefault
+                    x.isPublic = temp[0].isPublic
+                }
+                res.push(x)
+                return res
+            }, [])
+            const idx = result.findIndex((i) => i.isDefault)
+            if (idx !== -1) {
+                result.unshift(result.splice(idx, 1)[0])
+            } else {
+                result[0].isDefault = 1
+            }
+
+            setRawWalletList(result)
+            setRawPatchData(result)
+            return
+        }
+        walletsList.forEach((x, idx) => {
+            x.isPublic = 1
+            x.isDefault = 0
+            if (idx === 0) {
+                x.isDefault = 1
+                return
+            }
+        })
+        setRawWalletList(cloneDeep(walletsList))
+        setRawPatchData(cloneDeep(walletsList))
+    }, [proofRes, kv, bodyView])
+
+    const onCancel = () => {
+        setRawPatchData(cloneDeep(rawWalletList))
+        setHasChanged(false)
+    }
+
+    const refresh = () => {
+        setBodyView(BodyViewSteps.main)
+        retryProof()
+        retryKv()
+    }
+    const WalletButton = () => {
+        const { classes } = useStyles()
+        if (bodyView === BodyViewSteps.addWallet) return null
+        return (
+            <div className={classes.btnContainer}>
+                <Button
+                    onClick={() => {
+                        setBodyView(
+                            bodyView === BodyViewSteps.wallets ? BodyViewSteps.addWallet : BodyViewSteps.wallets,
+                        )
+                    }}
+                    className={classes.walletBtn}
+                    variant="contained"
+                    size="small">
+                    {bodyView === BodyViewSteps.wallets ? BodyViewSteps.addWallet : BodyViewSteps.wallets}
```

Comment:
Should not display enum value on UI.
In general, enums are for key identifying, but no for UI text.
Need i18n here too.